bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx135-na.html

```diff
@@ -0,0 +1,76 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_135_pdf') }}
+{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main mzp-l-content mzp-t-content-md">
+    <h1 class="wnp-main-title">
+      {{ picture(
+        url="img/firefox/whatsnew/whatsnew135-pdf/edit-pdfs-light.svg",
+        sources=[
+            {
+                "media": "(prefers-color-scheme: light)",
+                "srcset": {
+                    "img/firefox/whatsnew/whatsnew135-pdf/edit-pdfs-light.svg": "default",
+                }
+            },
+            {
+                "media": "(prefers-color-scheme: dark)",
+                "srcset": {
+                  "img/firefox/whatsnew/whatsnew135-pdf/edit-pdfs-dark.svg": "default",
+              }
+            }
+        ],
+        optional_attributes={
+          "alt": "Edit PDFs in Firefox",
+          "width": "435",
+          "height": "222",
+        }
+    ) }}
+    </h1>
+
+    <div class="wnp-loading"></div>
+
+    <div class="wnp-editor">
+      <p class="wnp-main-tagline">
+        Say goodbye to the hassle of printing or filling out PDFs in other apps. Quickly and easily edit documents right in your browser.
+      </p>
+
+
+      <p class="wnp-main-cta">
+        <a class="mzp-c-button mzp-t-product" href="https://assets.mozilla.net/wnp124-en/wnp-fx124-en.pdf" data-cta-type="button" data-cta-text="Try it now" target="_blank" rel="noopener">
+          Try it now
+        </a>
+      </p>
+    </div>
+
+    <div class="wnp-default">
+      <p class="wnp-main-tagline">
+        Say goodbye to the hassle of printing or filling out PDFs in other apps. With Firefox as your default browser, quickly and easily edit documents right in your browser.
+      </p>
+
+      <p class="wnp-main-cta is-not-default">
+        <a class="mzp-c-button mzp-t-product" href="{{ url('firefox.set-as-default.thanks') }}" data-cta-type="button" data-cta-text="Set Firefox to default" target="_blank" rel="noopener">
```

Comment:
TODO: update to remove conditional Set to Default logic. This was carried over from previous WNP and is not needed in this iteration